lib/miq_automation_engine/engine/miq_ae_service_model_base.rb

```diff
@@ -29,7 +31,7 @@ def self.respond_to?(m, *args)
 
     # Expose the ActiveRecord find, all, count, and first
     def self.class_method_exposed?(m)
-      m.to_s.starts_with?('find_') || [:where, :find, :all, :count, :first].include?(m)
+      m.to_s.starts_with?('find_') || [:where, :find].include?(m)
```

Comment:
Not sure if you can just deprecate these. Do customers have automate scripts that call these? @gmcculloug ?

Think we may need the legacy finders stuff over here @matthewd 

Also.

We are exposing all the power of active record. Not sure this is so wise.
If we want to change the api here, we may want to make the method calls more structured and limited. Otherwise we're opening ourselves up to supporting everything rails has ever supported and anything they do in the future.
And to security vulnerabilities.

Or do we just view this as a bandaid until we get to rest.
(again, if we go to rest, we'll want to limit the api people call to fetch records)